cli/runner/cmd/run.go

```diff
@@ -51,7 +56,14 @@ var runCmd = &cobra.Command{
 			return
 		}
 
-		var cmdClient client.Requester = client.NewClient()
+		// add a flag to modify timeout?
+		var cmdClient client.Requester
+		clint := client.Config{
+			BaseUrl: url,
+			Timeout: coderunner.TIMEOUT_DEFAULT,
```

Comment:
Yeah I realized as I was doing it that maybe adding a future flag for allowing users to modify the timeout would be problematic; even with concurrent requests it would be a simple DOS if you could make the timeout arbitrary as a user.